File: day4/p1.cpp
#include<iostream>
using namespace std;

int main()
{
    int noOfRows,noOfCols;

    cout<<"Enter the number of rows and columns:";

    cin>>noOfRows>>noOfCols;

    for(int rows=1;rows<=noOfRows;rows++)
    {
        for(int cols=1;cols<=noOfCols;cols++)
        {
            cout<<"*";
        }
        cout<<endl;
    }
    return 0;
}

File: day4/p1.java
package Git.day4;

import java.util.Scanner;

public class p1{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter the Number of rows and colums : ");
        int noOfRows=sc.nextInt();
        int noOfCol=sc.nextInt();

        for(int row=1;row<=noOfRows;row++)
        {
            for(int col=1;col<=noOfCol;col++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}
